### ISSUE TITLE
design mobile navbar

### ISSUE THREAD

**Issue by guylepage3** [blockstack/blockstack.org#191]
Design a mobile navigation that solves the hamburger menu / 5 second bounce rule.

**Solutions**
- Limited display in combination with a greedy hamburger menu
- Display only high priority items (1 or 2 items)

**Comment by guylepage3** [comment]
Some great examples of the combination of limited nav items w/ a greedy hamburger menu are;

BBC
<img width="487" alt="screenshot 2017-01-09 11 51 07" src="https://cloud.githubusercontent.com/assets/1711854/21775295/80c8e75c-d663-11e6-9b05-8ad31b6c8f89.png">

The Guardian
<img width="486" alt="screenshot 2017-01-09 11 51 28" src="https://cloud.githubusercontent.com/assets/1711854/21775306/8c6f35ac-d663-11e6-8b7b-998f9d49d285.png">

**Comment by guylepage3** [comment]
Another example is Starbucks. Although not quite as effective as it uses icons.
<img width="473" alt="screenshot 2017-01-09 11 51 54" src="https://cloud.githubusercontent.com/assets/1711854/21775338/ab7a2d12-d663-11e6-84c7-825fad651903.png">

**Comment by guylepage3** [comment]
Also, here's a great Podcast I found over the weekend that provides ideas on how to solve the hamburger menu issue.

https://soundcloud.com/chris-myhill-1/my-beef-with-the-hamburger-menu

**Comment by guylepage3** [comment]
So I believe I came up with a solution for navigation on the landing view of blockstack.org mobile.

I placed all navigation within the natural heat map sweet spot for users thumbs.
See: https://cloud.githubusercontent.com/assets/1711854/21835716/3a0a99ca-d78d-11e6-93a4-a94ddcbbcfb6.png

This allows users to see all nav items, unless they are on a smaller device than an iPhone 6. As the device size shrinks the header nav items pop off from bottom to top.

<img width="456" alt="screenshot 2017-01-10 23 26 44" src="https://cloud.githubusercontent.com/assets/1711854/21855068/0fc07fc8-d7eb-11e6-90d0-714c5195a4d5.png">
